.buildkite/gpu-tests/scripts/run-tests.sh

```
@@ -18,10 +18,15 @@ echo "~~~ GPU info"
 nvidia-smi
 
 echo "--- Setting up Python environment"
+BUILD_DIR="/model-cache/.builds/${BUILDKITE_BUILD_ID}"
+rm -rf "$BUILD_DIR"
+mkdir -p "$BUILD_DIR"
+trap "rm -rf $BUILD_DIR" EXIT
```

Shellcheck (0.11.0)

[warning] Use single quotes, otherwise this expands now rather than when signalled.

(SC2064)